app/classes/pivotal/http.rb

```diff
@@ -8,17 +9,25 @@ class Pivotal
 
   class << self
     def get_stories(_verbose = false)
-      json = get_request("stories?limit=10000&#{story_fields}")
       stories = []
+
+      request_stories("", stories)
+      request_stories("&offset=501", stories) if stories.count == 500
```

Comment:
Hey, so what's the approx. number of stories currently?  Do you happen to know off the top of your head?  Now that you're removing accepted stoires, etc. maybe it's not so prohibitively many?